src/test/kotlin/com/atlassian/performance/tools/infrastructure/os/PidstatIT.kt

```diff
@@ -0,0 +1,29 @@
+package com.atlassian.performance.tools.infrastructure.os
+
+import com.atlassian.performance.tools.infrastructure.api.os.Ubuntu
+import com.atlassian.performance.tools.infrastructure.toSsh
+import com.atlassian.performance.tools.sshubuntu.api.SshUbuntuContainer
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+
+class PidstatIT {
+    @Test
+    fun shouldProduceLogFile() {
+        SshUbuntuContainer.Builder()
+            .version("20.04")
```

Comment:
The test doesn't seem to be 20.04-specific.
```suggestion
            .version("20.04")
```